modules/openapi-generator/src/main/resources/python/model_templates/methods_shared.mustache

```diff
@@ -12,4 +12,27 @@
 
     def __getattr__(self, attr):
         """get the value of an attribute using dot notation: `instance.attr`"""
-        return self.{{#attrNoneIfUnset}}get{{/attrNoneIfUnset}}{{^attrNoneIfUnset}}__getitem__{{/attrNoneIfUnset}}(attr)
\ No newline at end of file
+        return self.{{#attrNoneIfUnset}}get{{/attrNoneIfUnset}}{{^attrNoneIfUnset}}__getitem__{{/attrNoneIfUnset}}(attr)
+
+    def __copy__(self):
+        cls = self.__class__
+        if self.get("_spec_property_naming", False):
+            return cls._new_from_openapi_data(**self.__dict__)
+        else:
+            return new_cls.__new__(cls, **self.__dict__)
```

Comment:
Do we need to call init?

Reply:
Doesn't the __new__ definition already do that? 

https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/main/resources/python/model_utils.mustache#L227